pennylane/templates/embeddings.py

```diff
@@ -164,56 +181,48 @@ def BasisEmbedding(features, wires):
     wires, n_wires = _check_wires(wires)
     _check_shape(features, (n_wires,))
 
-    # basis_state cannot be trainable
-    msg = "The input features in BasisEmbedding influence the circuit architecture and can " \
-          "therefore not be passed as a positional argument to the quantum node."
-    _check_no_variable([features], ['features'], msg=msg)
-
     # basis_state is guaranteed to be a list
     if any([b not in [0, 1] for b in features]):
         raise ValueError("Basis state must only consist of 0s and 1s, got {}".format(features))
     ###############
 
     features = np.array(features)
-
     BasisState(features, wires=wires)
 
 
-def SqueezingEmbedding(features, wires, method='amplitude', c=0.1):
-    r"""Encodes :math:`N` features into the squeezing amplitudes :math:`r \geq 0` or phases :math:`\phi \in [0, 2\pi)`
-    of :math:`M` modes, where :math:`N\leq M`.
+def DisplacementEmbedding(features, wires, method='amplitude', c=0.1):
+    r"""Encodes :math:`N` features into the displacement amplitudes :math:`r` or phases :math:`\phi` of :math:`M` modes,
+     where :math:`N\leq M`.
```

Comment:
This is very confusing (Squeezing->Displacement), but I'm assuming github messed up the diff?

Reply:
How come you switched the orders of the templates?

Reply:
Ah, your review answers my question!